aries-site/src/components/home/CreativeToolkit.js

```diff
@@ -0,0 +1,90 @@
+import React, { useContext } from 'react';
+// eslint-disable-next-line import/no-unresolved
+import * as tokens from 'hpe-design-tokens/docs';
+import {
+  Box,
+  Heading,
+  Image,
+  PageContent,
+  ResponsiveContext,
+} from 'grommet';
+import { ContentPreviewCard } from '../cards';
+import { useDarkMode } from '../../utils';
+
+import { toolkitItems } from '../../data/toolkitItems';
+
+const CreativeToolkitLayout = () => {
+  const size = useContext(ResponsiveContext);
+  const darkMode = useDarkMode();
+
+  return (
+    <Box
+      direction={!['xsmall', 'small'].includes(size) ? 'row' : 'column'}
+      gap="large"
+      justify="center"
+      align="center"
+    >
+      {toolkitItems.map(({ name, href, image }) => (
+        <ContentPreviewCard key={name} href={href} pad="medium" round="xxlarge">
+          <Box
+            align="center"
+            justify="center"
+            gap="medium"
+            height="small"
+            width={!['xsmall', 'small'].includes(size) ? 'medium' : 'large'}
+          >
+            <Box
+                width="xxsmall"
+                height="xsmall"
+                align="center"
+                justify="center"
+            >
+              {image && (
+                <Image
+                    src={
+                      darkMode.value
+                        ? image.src.dark || image.src
+                        : image.src.light || image.src
+                    }
+                  alt={image.alt}
+                  fit="contain"
+                  // Use individual width or fallback
+                  width={image.width || '128px'} 
```

Comment:
Use tokens. No hardcoded values.

```suggestion
                  width={image.width || 'xxsmall'} 
```